src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2782,3 +2782,8 @@ Warning\:\ The\ selected\ directory\ is\ not\ a\ valid\ directory.=Warning: The
 Currently\ selected\ JStyle\:\ '%0' = Currently selected JStyle: '%0'
 Currently\ selected\ CSL\ Style\:\ '%0' = Currently selected CSL Style: '%0'
 Store\ url\ for\ downloaded\ file=Store url for downloaded file
+
+BibTeX\ of\ that\ entry=BibTeX of that entry
```

Comment:
Re-use "Show BibTeX source" (or "Show %0 source". Just check the existing strings.)

Reply:
Exits only "Show/edit %0 source"